internal/pkg/server/fleet_integration_test.go

```diff
@@ -1485,17 +1485,6 @@ func Test_SmokeTest_AuditUnenroll(t *testing.T) {
 	require.Equal(t, http.StatusOK, res.StatusCode)
 	res.Body.Close()
 
-	t.Log("Orphaned can replace uninstall")
```

Comment:
We need to test/support this scenario.
It's important for Endpoint to be able to signal that it is orphaned even if the installed agent is removed